packages/slice-machine/server/src/api/slices/push.ts

```diff
@@ -44,26 +45,23 @@ export async function pushSlice(
       variations,
     };
 
-    const res = await createOrUpdate(
-      slices,
-      sliceName,
-      modelWithImageUrl,
-      env.client
-    );
-
-    if (res.status > 209) {
```

Comment:
why 209?

Reply:
haha, more than 2xx is considered errors I guess 😅

Reply:
We can always check the readme of the Custom type API, it is really precise.